### ISSUE TITLE
How to modify the optimizer to achieve gradient accumulation

### ISSUE THREAD

**Issue by adjawdka** [open-mmlab/mmdetection#12280]
When reproducing the code, the training accuracy is slightly lower than the paper, and the GPU does not support large batch sizes. How to set gradient accumulation.
